### PR TITLE
refine elasticdl:ci dockerfile for integration test

### PR DESCRIPTION
**Pull request by QiJune**
We need to make running integration test in Travis is the same as running in local.

We need a Dockerfile like this:

```
FROM tensorflow
RUN pip install elasticdl 
COPY model_zoo /model_zoo
```

This PR will do the first step, we make a Dockerfile for ci like following:

```
FROM tensorflow

COPY --from=dev /elasticdl-develop-py3-none-any.whl /
RUN python -m pip -q install /elasticdl-develop-py3-none-any.whl \
        --extra-index-url=${EXTRA_PYPI_INDEX} \
    && rm /elasticdl-develop-py3-none-any.whl

// to be simplified in the next step
COPY --from=dev ${GO_PS} /usr/local/go/bin

COPY model_zoo /model_zoo

// to be simplified in the next step
COPY --from=dev /data /data
```

So, in the integration test, we could directly pass "elasticdl:ci" to `image_name`  to launch a training job.

### CODE REVIEW

File: elasticdl/docker/build_all.sh [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
 
 TF_VERSION=2.1.0
 
 if [[ ! -d .git ]]; then
     echo "We must run this script at the root of the source tree."
-    exit -1
```
Comment (wangkuiyi) [inline elasticdl/docker/build_all.sh:7]: It seems that we cannot remove the `exit` command?

File: elasticdl/docker/build_all.sh [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
 fi
 
 if [[ $# -eq 1 && $1 == "-gpu" ]]; then
     base_image="tensorflow/tensorflow:${TF_VERSION}-gpu-py3"
     echo "To support CUDA; all images are from " $base_image
-    image="elasticdl:gpu"
     dev_image="elasticdl:gpudev"
```
Comment (wangkuiyi) [inline elasticdl/docker/build_all.sh]: elasticdl:dev-gpu
elasticdl:ci-gpu

File: elasticdl/python/common/save_utils.py [hunk 3]
```diff
@@ -1,3 +1,16 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
```
Comment (wangkuiyi) [inline elasticdl/python/common/save_utils.py:1]: It seems that in the next PR, we should remove the filename from `save_utils.py` into `checkpoint.py`.

File: elasticdl/docker/build_all.sh (hunk 1 above)
Comment (QiJune) [reply elasticdl/docker/build_all.sh:7]: Yes, fix it. 

Add `exit 1`

File: elasticdl/docker/build_all.sh (hunk 2 above)
Comment (QiJune) [reply elasticdl/docker/build_all.sh]: Done.

File: elasticdl/python/common/save_utils.py (hunk 3 above)
Comment (QiJune) [reply elasticdl/python/common/save_utils.py:1]: Exactly, `checkpoint.py` is more meaningful than `save_utils.py`

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**